__tests__/app.test.js

```diff
@@ -35,7 +36,51 @@ afterAll(() => db.end());
             .get('/api')
             .expect(200)
             .then(({body}) => {
-                expect(body).toEqual(endPoints)    
+                expect(body.endpoints).toEqual(endPoints)    
+            })
+        })
+    })
+
+    describe('/api/articles', () => {
+        it('should get an article by _Id and respond with the corresponding article object with the correct properties', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({body}) => {
+                const article = body.article
+                console.log(article)
+                expect(article.length).toEqual(1)
+                article.forEach((element) => {
+                    expect(element).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            body: expect.any(String),
+                            topic: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            article_img_url: expect.any(String),
```

Comment:
you should assert against the specific article instead of these generic marchers here to make your tests stricter

Reply:
resolved :)